test/integration/api/projects.js

```diff
@@ -359,12 +365,34 @@ describe('api: /projects', () => {
           .expect(200)
           .then(({ body }) => {
             body.verbs.should.be.an.Array();
-            body.verbs.length.should.be.lessThan(26);
-            body.verbs.should.containDeep([ 'assignment.create', 'project.delete' ]);
+            body.verbs.length.should.be.lessThan(30);
```

Comment:
Since you ended up adding 2 verbs instead of 4, should this be 28 instead of 30?